zebra-network/src/protocol/external/types.rs

```diff
@@ -106,8 +106,10 @@ impl Version {
             (Mainnet, Nu5) => 170_100,
             (Testnet(params), Nu6) if params.is_default_testnet() => 170_110,
             (Mainnet, Nu6) => 170_120,
-            (Testnet(params), Nu7) if params.is_default_testnet() => 170_130,
-            (Mainnet, Nu7) => 170_140,
+            (Testnet(params), Nu6_1) if params.is_default_testnet() => 170_130,
```

Comment:
These should also handle regtest as a testnet.

Reply:
This is being addressed in https://github.com/ZcashFoundation/zebra/pull/9710.